hw/system/snitch_cluster/cfg/cluster.default.hjson

```diff
@@ -62,10 +69,10 @@
         ]
     },
     dram: {
-        // 0x8000_0000
-        address: 2147483648,
-        // 0x8000_0000
-        length: 2147483648
+        // 0x9000_0000
```

Comment:
Kind of, `snrt_global_memory()` is initialized with those values. If it would start at `0x8000_0000` you would potentially overwrite the `.text` section, if `snrt_global_memory()` is used for allocating data.

Reply:
Okay, but you could add the `0x1000_0000` offset in `snrt_global_memory()` as well, right? Seems kind of cheating to artificially reduce the memory in the system description (which should be a true and holistic view of the system).

Reply:
True, I will change this accordingly